src/main/webapp/css/main.css

```diff
@@ -39,6 +39,11 @@ nav a {
   font-size: 36px;
 }
 
+img {
```

Comment:
nit: The indentation is a bit off. Make sure it matches everything else on the file.